### DISCUSSION TITLE
Dịch Scaled Dot-Product Attention

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#155] [Terminology] [answered]
Bình thường từ "scale" hay được dịch theo glossary là "co giãn", ý nói nhân thêm hệ số với một đại lượng nào đó để thay đổi khoảng giá trị của đại lượng này.

Nếu dịch "Scaled Dot-Product Attention" là "Tập trung Tích vô hướng Co giãn" thì nghe có vẻ hơi khó hiểu, mọi người có giải pháp gì không nhỉ?

@mlbvn/handson-ml

**Comment by thanhcsf** [comment]
Mình chưa nghĩ ra từ gì khác, không biết hai anh có gợi ý gì không ạ? @ngcthuong @tiepvupsu

**Reply by ngcthuong** [reply]
Mình ko tìm được từ khác.

**Comment by lkhphuc** [comment]
Scaled tính từ mình dịch là `tỷ lệ`? `Tập trung Tích vô hướng (theo) tỷ lệ`.
Trong ngữ cảnh khác thì có như: Bản đồ có scale /  tỷ lệ là 1:100... chẳng hanh.

**Reply by thanhcsf** [reply]
option này dễ hiểu hơn `co giãn`

**Reply by ngcthuong** [reply]
Tỷ lệ là danh từ, còn đây là động từ mà.

**Reply by lkhphuc** [reply]
Thế nên em em thấy `...theo tỷ lệ` mới là chuẩn nhất mà sợ dài quá.

**Reply by cuongvng** [reply]
Em vote cho `Tập trung Tích vô hướng theo tỷ lệ`

**Comment by thanhcsf** [comment, marked as answer]
Mình tạm chốt theo số biểu quyết hiện có là `Tập trung Tích vô hướng theo tỷ lệ` nhé.
